AdvLoggerPkg/Library/AssertTelemetryLib/AssertLib.c

```diff
@@ -86,7 +86,7 @@ DebugAssert (
   //
   // Send the print string to the Logging device device
   //
-  AdvancedLoggerWrite (DEBUG_ERROR, Buffer, AsciiStrLen (Buffer));
+  AdvancedLoggerWrite (DEBUG_ERROR, Buffer, AsciiStrnLenS (Buffer, sizeof (Buffer)Buffer));

```

Comment:
Was the `Buffer` after `sizeof()` meant to be there?

```suggestion
  AdvancedLoggerWrite (DEBUG_ERROR, Buffer, AsciiStrnLenS (Buffer, sizeof (Buffer)));

```

Reply:
Thanks for catching it.